css-canvas-medical/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row1">
+      <div class="logo">
+        <img class="imagecanvas-top" src="images/logo-blue.svg" alt="canvas">
+        <nav class="navigation">
+          <a href="" class="nav-items">Product</a>
+          <a href="" class="nav-items">Implementation</a>
+          <a href="" class="nav-items">About Us</a>
+          <a href="" class="nav-items">Blog</a>
+          <a href="" class="nav-items" id="bordered-nav">Contact Us</a>
+        </nav>
+      </div>
+    </div>
+    <div>
+      <hr class="header-line">
+    </div>
+    <div class="row2">
+      <div class="hero-image">
+        <div class="overlay"></div>
+        <img class="hero" src="images/hero.jpg" alt="hero">
+        <h2>The EMR Reimagined for Primary Care</h2>
+        <p class="paragraph-image">Can't remember the last time you talked with a patient without focusing
+          on the EMR? Canvas lets you rediscover joy in the work you do. Capture
+          meaningful data, identify gaps in care, and move seemlessly between
+          tasks within the platform. You stay present with patients while your
+          whole care team stays up-to-speed. Be a better doctor with Canvas,
+          designed for primary care providers with the help of primary care
+          providers.
+        </p>
+        <a href="" class="bordered-nav2">Get Started</a>
+      </div>
+    </div>
+    <div class="row3">
+      <div class="body1">
```

Comment:
More numbered sections, this is not a great way to name classes. A class name should always be descriptive, either of its purpose if it's a utility class, or the element it's being applied to if it's a semantic class. Numbered versions of classes give us no knowledge of their purpose.